vividus-plugin-rest-api/src/test/java/org/vividus/bdd/steps/api/JsonResponseValidationStepsTests.java

```diff
@@ -105,6 +106,9 @@
     private static final String JSON_PATH = "$..value";
     private static final String THE_NUMBER_OF_JSON_ELEMENTS_ASSERTION_MESSAGE =
             "The number of JSON elements by JSON path: ";
+    private static final String HTML =
+            "<html><head><meta http-equiv=\"Content-Type\" content=\"text/html;charset=utf-8\"/>";
+    private static final String PT_1_S = "PT1S";
```

Comment:
`PT_1_S ` -> ONE_SECOND ? and `Duration.ofSeconds(1)` ?

Reply:
> `PT_1_S ` -> ONE_SECOND ? and `Duration.ofSeconds(1)` ?
Updated